client/src/frontend/components/composed/EntityHero/parts/Masthead/styles.js

```diff
@@ -0,0 +1,26 @@
+import styled from "@emotion/styled";
+import { transientOptions } from "helpers/emotionHelpers";
+
+export const Wrapper = styled("div", transientOptions)`
+  max-height: 300px;
+  min-height: 180px;
```

Comment:
It's not clear to me what would cause this to ever be more than 180px tall since the image uses `object-fit` and the logo is absolutely positioned. I tested it out with a few different logos and that seemed to confirm as much. So some additional logic seems necessary to match the comps. Let me know if you want to discuss.